pyproject.toml

```diff
@@ -32,6 +32,11 @@ authors = [""]
 python = ">=3.8"
 ops = ">=2.0.0"
 kazoo = ">=2.8.0"
+
+# The cosl dep could be removed from here once PYDEPS is released:
+# https://github.com/canonical/charmcraft/pull/1003
```

Comment:
**question:** I believe this has merged. What would need to be updated here?

Reply:
The change was merged but it is not in stable (and possible broken now).
`stable` is still 2.2.0, and PYDEPS isn't there.

With the next stable release of charmcraft you should drop this line from the deps. Charmcraft will automatically pick it up [from the lib](https://github.com/canonical/grafana-agent-k8s-operator/blob/884b687702be5c147fe2a081603ec819f1fe69ce/lib/charms/grafana_agent/v0/cos_agent.py#L181).